frame/composable-traits/src/dex.rs

```diff
@@ -375,6 +381,15 @@ pub struct PriceAggregate<PoolId, AssetId, Balance> {
 	pub spot_price: Balance, // prices based on any other stat such as TWAP goes here..
 }
 
+/// RedeemableAssets for given amount of lp tokens.
+#[derive(RuntimeDebug, Encode, Decode, Default, Clone, PartialEq, TypeInfo)]
+#[cfg_attr(feature = "std", derive(Serialize, Deserialize))]
+pub struct RedeemableAssets<Balance> {
+	pub lp_tokens: Balance,
```

Comment:
My original suggestion was something like this, so that we can expand the number of assets and their balances returned without breaking clients.
```suggestion
	pub BTreeMap<AssetId, Balance> assets;
```
Can remove all the other fields here as well as knowing the base/quote asset is not important when redeeming